src/browser_app_skeleton/src/components/shared/layout_head.cr

```diff
@@ -10,6 +10,7 @@ class Shared::LayoutHead < BaseComponent
       meta name: "turbolinks-cache-control", content: "no-cache"
       csrf_meta_tags
       responsive_meta_tag
+      live_reload_connect_tag
```

Comment:
Maybe this could have a small comment on what it's for? Like
```
# Used in development to reload your browser
# See [whatever docs]()
```

Just an example, but whatever you think might be clear so newcomers will know.